src/Microsoft.SqlTools.SqlCore/ObjectExplorer/Nodes/INodeFilter.cs

```diff
@@ -13,23 +13,23 @@ namespace Microsoft.SqlTools.SqlCore.ObjectExplorer.Nodes
     /// <summary>
     /// Has information for filtering a SMO object by properties 
     /// </summary>
-    public interface INodeFilter
+    public abstract class INodeFilter
```

Comment:
Why is this an abstract class? I don't see any implementation here.

Reply:
It does implement static methods which are not allowed for an interface in lang version 8

Reply:
Ah, that's unfortunate.

Almost feels like it would have been easier/better to just update our workload to net6...going forward we should be trying to avoid using netfx whenever possible. Anyways too late for that now I guess.

Reply:
I agree. We should move other projects to net core 7/8. However, that will be a bigger task.